kasa/tests/test_protocol.py

```diff
@@ -64,6 +65,35 @@ def aio_mock_writer(_, __):
     assert response == {"great": "success"}
 
 
+@pytest.mark.skipif(sys.version_info < (3, 8), reason="3.8 is first one with asyncmock")
+async def test_protocol_logging(mocker, caplog):
```

Comment:
This should probably be parametrized to test it on both logging.WARNING and logging.DEBUG?

Reply:
Will do